source/getting-started/onboarding.markdown

```diff
@@ -3,56 +3,53 @@ title: "Onboarding Home Assistant"
 description: "Instructions to get Home Assistant configured."
 ---
 
-Alright, you made it here. The tough part is done.
+After your Home Assistant server is up and running, there are 5 simple steps to log in and see your devices.
```

Comment:
- I don't think we should use the term "server" to be honest.
- Let's avoid the use of "simple", which is subjective and can be received as condescending in case a user is struggling.
- I'm unclear on "and see your devices" and what that means?

Reply:
what would you prefer instead of server? "See your devices is a phrase used in the previous version so I assumed it was that approved way of discussing this?

Reply:
Given your previous reply, I can change this to "After Home Assistant is up and running..."